pkgs/games/exult/default.nix

```diff
@@ -12,35 +12,30 @@ let
 in
 
 stdenv.mkDerivation rec {
-  name = "exult-1.4.9rc1";
+  name = "exult-1.5.0git";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/exult/${name}.tar.gz";
-    sha256 = "0a03a2l3ji6h48n106d4w55l8v6lni1axniafnvvv5c5n3nz5bgd";
+  src = fetchFromGitHub {
+    owner = "exult";
+    repo = "exult";
+    rev = "b727abfffc08a54e528bc1194d0de8562d18d74e";
```

Comment:
So what's up with their releases names: https://github.com/exult/exult/releases ?

It seems to me they use the releases names to imply what major changes were done in that release. Hence, I'd switch to a date based `version = ` and download whatever they count as a release.